validation/src/required.ts

```diff
@@ -23,7 +23,7 @@ import { ValidationErrors } from 'ngrx-forms';
  * ```
  */
 export function required<T>(value: T | null): ValidationErrors {
```

Comment:
To support strict type checking you need to make it explicit that the function accepts `undefined` as a value, i.e.

```typescript
export function required<T>(value: T | null | undefined): ValidationErrors {
```

This will also fix the TSLint error.